main/curl/APKBUILD

```diff
@@ -101,12 +104,11 @@ build() {
 }
 
 check() {
-	cd "$builddir"
-	make -C tests nonflaky-test
+	cd "$builddir"/tests
+	make nonflaky-test
 }
 
 package() {
-	cd "$builddir"
```

Comment:
This is a package in main, ncopa asked to leave these in main. Especially for backporting, you want to introduce as less noise as possible.